src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityService.kt

```diff
@@ -86,7 +85,7 @@ class EligibilityService(
   }
 
   private fun hasReleaseDateInTheFuture(): EligibilityCheck = early@{
-    val releaseDate = it.getLicenceStartDate() ?: return@early true
+    val releaseDate = it.conditionalReleaseDate ?: return@early true
```

Comment:
Calculating licence start date here caused at least one extra query to NOMIS per case (n queries). As such, it was decided to just check CRD, as we get this for free and should represent eligibility well enough (in combination with the other rules)